demo-simple/src/main/java/com/novoda/downloadmanager/demo/LandingActivity.java

```diff
@@ -0,0 +1,29 @@
+package com.novoda.downloadmanager.demo;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+public class LandingActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_landing);
+
+        View migrationView = findViewById(R.id.migration);
```

Comment:
same, would inline here and extract a common method with just the Activity class that you want to start, two lines of code.

Reply:
7bfea1a